.github/workflows/pythonpackage.yml

```diff
@@ -44,8 +44,10 @@ jobs:
           TEST_DATABASE: ${{ secrets.TEST_DATABASE }}
           TEST_BLOB: ${{ secrets.TEST_BLOB }}
           ENGINE_CONNECTION_STRING: ${{ secrets.ENGINE_CONNECTION_STRING }}
+          APPLICATION_INSIGHTS_ENGINE_CONNECTION_STRING: ${{https://ade.applicationinsights.io/subscriptions/12534eb3-8109-4d84-83ad-576c0d5e1d06/resourcegroups/clients_e2e_test/providers/microsoft.insights/components/kusto-e2e-app-insights}}
```

Comment:
The ${{}} syntax is for variables from github.
Since we hardcode them here, you don't need the ${{}}, just write the values. (That's why your tests don't run automatically)

Reply:
fixed